packages/vue-components/src/Navbar.vue

```diff
@@ -254,34 +239,98 @@ export default {
         if (!content.contains(e.target)) content.classList.remove('open');
       });
     });
-    $(this.$el).on('click', 'li:not(.dropdown)>a', (e) => {
-      if (e.target.classList.contains('submenu-toggle')) { return; }
-      setTimeout(() => { this.collapsed = true; }, 200);
-    }).onBlur((e) => {
-      if (!this.$el.contains(e.target)) { this.collapsed = true; }
-    });
-    if (this.slots.collapse) $('[data-toggle="collapse"]', this.$el).on('click', e => this.toggleCollapse(e));
 
     // highlight current nav link
     this.highlightLink(window.location.href);
 
     this.toggleLowerNavbar();
     $(window).on('resize', this.toggleLowerNavbar);
+
+    // scroll default navbar horizontally when mousewheel is scrolled
+    const navbarDefault = this.$el.querySelector('.navbar-default');
+    $(navbarDefault).on('wheel', (e) => {
+      const isDropdown = (nodes) => {
+        for (let i = 0; i < nodes.length; i += 1) {
+          if (nodes[i].classList && nodes[i].classList.contains('dropdown-menu')) {
+            return true;
+          }
+        }
+        return false;
+      };
+
+      // prevent horizontal scrolling if the scroll is on dropdown menu
+      if (window.innerWidth < 768 && !isDropdown(e.path)) {
+        e.preventDefault();
+        navbarDefault.scrollLeft += e.deltaY;
+      }
+    });
   },
   beforeDestroy() {
     $('.dropdown', this.$el).off('click').offBlur();
-    if (this.slots.collapse) $('[data-toggle="collapse"]', this.$el).off('click');
     $(window).off('resize', this.toggleLowerNavbar);
+    $(this.$el.querySelector('.navbar-default')).off('wheel');
   },
 };
 </script>
 
 <style scoped>
     @media (max-width: 767px) {
-        .navbar-collapse {
-            max-height: 80vh !important;
-            overflow-x: hidden !important;
-            overflow-y: scroll !important;
+        .navbar {
+            padding-left: 0;
+            padding-right: 0;
+            padding-bottom: 0;
+        }
+
+        div.navbar-brand {
+            padding-left: 1rem;
+        }
+
+        .navbar-right {
+            position: absolute;
+            top: 14px;
+            right: 16px;
+            max-width: 40vw;
+        }
+
+        .navbar-default {
+            display: block;
+            margin-top: 0.3125rem;
+            width: 100%;
+            overflow-x: scroll;
+
+            /* Hide overflow scroll bar */
+            -ms-overflow-style: none;  /* IE and Edge */
+            scrollbar-width: none;  /* Firefox */
+        }
+
+        /* Hide overflow scroll bar for Chrome and Safari */
+        .navbar-default::-webkit-scrollbar {
+            display: none;
+        }
+
+        .navbar-default ul {
+            flex-direction: row;
+            margin-top: 0 !important;
+            width: 100%;
+        }
+
+        .navbar-default > ul > li,
+        .navbar-default > ul > div {
+            padding: 0.3125rem 0.625rem;
+            flex: 1;
+            background: rgba(0, 0, 0, 0.3);
```

Comment:
could be moved to `.navbar-default`, so items can set their own backgrounds

Reply:
Hmmm I was thinking that if a link on the navbar is highlighted, its background should also change to the main colour of the navbar. If we move this to `.navbar-default`, then I don't think we can do the background highlighting for the individual link. Any suggestion?

![image](https://user-images.githubusercontent.com/47470981/145839140-e4398162-5d57-4ff3-9d49-5a96e0354609.png)

Reply:
> The alternative I had in mind was to move the said color to .navbar-default, then forward the navbar main colour to .current specifically. This means forwarding .navbar-light/dark/primary's colours to the .current element manually.

I've now shifted the background overlay to the default navbar instead of the individual link. I've also forwarded the navbar colour to a css variable for `.current` to use.

> One slight issue with 0.3 specifically though - you'll likely need to try out / set a different value for .navbar-light, 0.3 seems a little dark for it.

Yup agree with this. I've decreased the overlay for primary and dark to `0.2` and `0.1` for light. Should be better now.

Primary (`0.2`):
![image](https://user-images.githubusercontent.com/47470981/146646323-4be40dae-7fbc-4d82-abea-b5312cfdbb88.png)

Dark (`0.2`):
![image](https://user-images.githubusercontent.com/47470981/146646315-6209a0e0-33be-4d7e-8870-244f1eb13c05.png)

Light (`0.1`):
![image](https://user-images.githubusercontent.com/47470981/146646300-cd18daea-0f1c-43ae-8900-05aa4b6dc172.png)